packages/capacitor/package.json

```diff
@@ -22,7 +22,7 @@
     "tslib": "^2.3.0"
   },
   "peerDependencies": {
-    "@nx/devkit": "^19.0.0",
-    "nx": "^19.0.0"
+    "@nx/devkit": "20.0.0",
```

Comment:
I think we should be compatible with `^20.0.0` to include further patches and minors.

Reply:
nice catch @edbzn 👍 will change

Reply:
done